lib/pages/submission_page.dart

```diff
@@ -0,0 +1,234 @@
+  }
+
+  void submitItem() async {
+    if (loading) {
+      setError("Form already submitted");
+    } else if (!formKey.currentState!.validate()) {
+      setError("Please fill in form fields");
+    } else if (bytes == null) {
+      setError("Invalid or no image uploaded");
+    } else {
+      setState(() {
+        loading = true;
+        error = "";
+      });
+      bool success = await uploadImageToFirebase("$itemName/$imageName");
+      if (success) {
+        await addFound();
+      }
+      routeBack(context);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final AppTheme theme = Theme.of(context).extension<AppTheme>()!;
+    final BoxDecoration bodyDecoration = BoxDecoration(
+      color: theme.veryLight,
+      borderRadius: const BorderRadius.all(Radius.circular(10)),
+    );
+    itemNameOptions.sort((a, b) => a.compareTo(b));
+
+    Widget buildError() =>
+        Text(error, style: theme.errorStyle, textAlign: TextAlign.center);
+
+    Widget buildHeader() => Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Text("Submit a found item", style: theme.darkTitleStyle),
+            ),
+            IconButton(
+                color: theme.dark,
+                icon: const Icon(Icons.keyboard_return_rounded),
+                onPressed: () => routeBack(context))
+          ],
+        );
+
+    Widget buildForm() => FastForm(
+            inputDecorationTheme: InputDecorationTheme(
+              disabledBorder: const OutlineInputBorder(
+                borderSide: BorderSide(width: 3),
+              ),
+              enabledBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.medium, width: 3),
+              ),
+              focusedBorder: const OutlineInputBorder(
+                borderSide: BorderSide(width: 3),
+              ),
+              errorBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.error, width: 3),
+              ),
+              focusedErrorBorder: OutlineInputBorder(
+                borderSide: BorderSide(color: theme.error, width: 2),
+              ),
+              filled: true,
+              fillColor: theme.light,
+            ),
+            formKey: formKey,
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: FastDropdown(
+                  initialValue: "Other",
+                  name: "item_name_dropdown",
+                  items: itemNameOptions,
+                  onChanged: (value) => setState(() => itemName = value!),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: FastTextField(
+                  maxLines: 5,
+                  name: "description_text_field",
+                  labelText: "Description",
```

Comment:
Can we make description labelText smaller? It's so jarring 😅

Reply:
![image](https://github.com/Steelhacks-2023/Lost-At-Pitt/assets/92108625/75eef028-22eb-408f-adc9-3d8b0aad74d6)
Is this preferable?

Reply:
I'm not able to make the reminder text in the box itself smaller with this widget.